pkg/config/project_config.go

```diff
@@ -161,5 +161,9 @@ func (c *ProjectConfig) GetPipeline(kind string) (*syntax.ParsedPipeline, error)
 	default:
 		return nil, fmt.Errorf("unknown pipeline kind %s", kind)
 	}
+
+	if parsed == nil {
+		return nil, fmt.Errorf("no pipeline defined for kind %s", kind)
```

Comment:
I'd use errors.Errorf for this in order to consistently use a single package for dealing with errors - `errors.New`, `errors.Errorf`, `errors.Wrap` and `errors.Wrapf`.

Reply:
Ok.